tests/test_urls.py

```diff
@@ -128,6 +132,29 @@ def test_new_issue_should_not_crash(self):
         rv = self.app.post('/issues/new', data=data)
         self.assertEqual(rv.status_code, 400)
 
+    def test_dashboard_triage(self):
+        """Request to /dashboard/triage should be 200."""
+        rv = self.app.get('/dashboard/triage')
+        self.assertEqual(rv.status_code, 200)
+        self.assertTrue('<h1>Triage Dashboard</h1>' in rv.data)
+        self.assertTrue('text/html' in rv.content_type)
+
+    def test_dashboard_route(self):
+        """Request to /dashboard should be 404.
+
+        For now, the dashboard route has no purpose.
+        """
+        rv = self.app.get('/dashboard/')
+        content_test = 'Lost in Punk Cat Space (404)' in rv.data
+        self.assertEqual(rv.status_code, 404)
+        self.assertTrue('text/html' in rv.content_type)
+        self.assertTrue(content_test)
+        rv = self.app.get('/dashboard')
```

Comment:
Is there any point in running these lines again? 
Accidental duplication?

Reply:
@miketaylr The first series is testing `/dashboard/` and then the second one testing `/dashboard` which are two different routes. My initial thought was to be sure to make 404 is the intent here. But many it's unnecessary as any unknown route is 404.

Reply:
ohhh, my bad. I didn't notice the trailing `/`. All good.